integration/internal/requirement/requirement_linux.go

```diff
@@ -0,0 +1,33 @@
+// +build !windows
```

Comment:
This can be removed (file already as `_linux.go` suffix)

Reply:
Done.